.github/workflows/go.yml

```diff
@@ -76,6 +83,7 @@ jobs:
           mkdir --parents /home/runner/go/src/github.com/vertgenlab/ &&
           cp -r ../gonomics /home/runner/go/src/github.com/vertgenlab/ &&
           cd /home/runner/go/src/github.com/vertgenlab/gonomics &&
+          go install ./... &&
           go test -coverprofile=coverage.txt -covermode=atomic ./...
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v2
```

Comment:
And if you upgrade to checkout@v3 and setup-go@v4, I'd check to make sure codecov-action@v2 is still compatible as well

Reply:
**actions/setup-go@v4** for lines:
- 16: https://github.com/vertgenlab/gonomics/blob/fcd91e13a696afbf717f942bcae27ce27eb20f69/.github/workflows/go.yml#L16
- 30: https://github.com/vertgenlab/gonomics/blob/fcd91e13a696afbf717f942bcae27ce27eb20f69/.github/workflows/go.yml#L30
- 47: https://github.com/vertgenlab/gonomics/blob/fcd91e13a696afbf717f942bcae27ce27eb20f69/.github/workflows/go.yml#L47
- 78: https://github.com/vertgenlab/gonomics/blob/fcd91e13a696afbf717f942bcae27ce27eb20f69/.github/workflows/go.yml#L78

actions/checkout@v3 for lines:
- 15: https://github.com/vertgenlab/gonomics/blob/fcd91e13a696afbf717f942bcae27ce27eb20f69/.github/workflows/go.yml#L15
- 29: https://github.com/vertgenlab/gonomics/blob/fcd91e13a696afbf717f942bcae27ce27eb20f69/.github/workflows/go.yml#L29
- 44: https://github.com/vertgenlab/gonomics/blob/fcd91e13a696afbf717f942bcae27ce27eb20f69/.github/workflows/go.yml#L44
- 75: https://github.com/vertgenlab/gonomics/blob/fcd91e13a696afbf717f942bcae27ce27eb20f69/.github/workflows/go.yml#L75